packages/testcontainers/src/wait-strategies/http-wait-strategy.test.ts

```diff
@@ -85,6 +109,42 @@ describe("HttpWaitStrategy", () => {
     ).rejects.toThrowError("URL /hello-world not accessible after 3000ms");
   });
 
+  it("should fail if container exited before healthcheck pass", async () => {
```

Comment:
Could we group these 2 tests and the `stopStartingContainer` function to a `describe` block, something like `when fail on container exit is true`

Reply:
Done